images/busybox/main.tf

```diff
@@ -0,0 +1,77 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+# Create another apko provider specifically for alpine, without the global wolfi defaults
+provider "apko" {
+  alias = "alpine"
+
+  default_archs = [] # this defaults to "all"
+  extra_keyring = ["https://dl-cdn.alpinelinux.org/alpine/edge/main"]
+}
```

Comment:
Ooh, nice. I suspect we'll want this in `tflib` eventually, so we can reuse it for git and static.

Reply:
I don't think that's possible, the alias needs to be defined by the caller (`module.latest`)

happy to be told otherwise!

Reply:
Can we try adding `apko_alpine = { source = { "chainguard-dev/apko" } }` above, and passing it in from the main module?